site/client/CookiePreferencesManager/CookiePreferencesManager.test.ts

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env yarn jest
+
+import {
+    PreferenceType,
+    parseRawCookieValue,
+    parseDate,
+    parsePreferences,
+    updatePreference,
+    getPreferenceValue,
+    serializeState,
+    arePreferencesOutdated,
+} from "./CookiePreferencesManager"
+
+describe("cookie preferences", () => {
```

Comment:
nit: when we just have 1 describe I've been dropping the "describe" and flattening the tree a bit.

Reply:
why not, but then you need to be more specific in your "it"? Or do you rely on the test file name to get context when a test fails?

What I like about the "describe" is that I don't have to go hunt for more context, I can just glance at the text in red and that's usually enough.

For example, this is pretty vague and in itself does not necessarily indicates where the test has failed (without looking for the offending test file name)...
<img width="290" alt="Screenshot 2020-10-13 at 14 27 06" src="https://user-images.githubusercontent.com/13406362/95860517-6e17b100-0d60-11eb-858b-7810c400caf5.png">

... whereas this in most cases can be self-sufficient
<img width="401" alt="Screenshot 2020-10-13 at 14 27 49" src="https://user-images.githubusercontent.com/13406362/95860630-9a333200-0d60-11eb-8c87-87be5131ffb9.png">

But again, you might be suggesting to drop the "describe" and have more specific "it".

Reply:
Ah, that's a good point I didn't consider. I guess most of the time lately I've been running tests 1 file at a time, so I already know the context and found the extra nesting unneccssary. You're right that when running the suite you'd have to manually go up and check what file a test was in.